js/colourpicker.js

```diff
@@ -369,129 +386,209 @@
 
       }
 
-      var hue, saturation, brightness, x, y, r, phi,
-
-      hex = input.val(),
+      var hue, saturation, brightness, x, y, r, phi, rgb, hsb, alpha, hex,
 
       // Helpful references
-      colourpicker = input.parent(),
+      colourpicker = input.parent();
       settings = input.data('colourpicker-settings'),
 
       // Panel objects
       grid = colourpicker.find('.colourpicker-grid'),
+      nnergrid = colourpicker.find('.colourpicker-grid-inner'),
```

Comment:
typo? Is this variable even used anywhere? something strange is happening because I do see references to `innergrid` but that variable is not defined here, yet it works. So many it's not even needed here, and it's defined elsewhere?